src/Scale/Color.elm

```diff
@@ -98,6 +403,33 @@ category10 =
     [ rgb255 31 119 180, rgb255 255 127 14, rgb255 44 160 44, rgb255 214 39 40, rgb255 148 103 189, rgb255 140 86 75, rgb255 227 119 194, rgb255 127 127 127, rgb255 188 189 34, rgb255 23 190 207 ]
 
 
+{-| A list of twenty categorical colors
+-}
+category20 : List Color
+category20 =
+    "1f77b4aec7e8ff7f0effbb782ca02c98df8ad62728ff98969467bdc5b0d58c564bc49c94e377c2f7b6d27f7f7fc7c7c7bcbd22dbdb8d17becf9edae5"
+        |> toHexColors
+        |> List.map hexToColor
+
+
+{-| A list of twenty categorical colors
+-}
+category20b : List Color
+category20b =
+    "393b795254a36b6ecf9c9ede6379398ca252b5cf6bcedb9c8c6d31bd9e39e7ba52e7cb94843c39ad494ad6616be7969c7b4173a55194ce6dbdde9ed6"
+        |> toHexColors
+        |> List.map hexToColor
+
+
+{-| A list of twenty categorical colors
+-}
+category20c : List Color
+category20c =
+    "3182bd6baed69ecae1c6dbefe6550dfd8d3cfdae6bfdd0a231a35474c476a1d99bc7e9c0756bb19e9ac8bcbddcdadaeb636363969696bdbdbdd9d9d9"
+        |> toHexColors
+        |> List.map hexToColor
+
+
```

Comment:
In [elm-visualization@2.0.0](https://github.com/gampleman/elm-visualization/releases/tag/2.0.0) we removed the category20 functions that were there:

> The scheme20... functions have been removed. They are not great for accessibility as many of the color combinations they produce are hard to distinguish.

They were implemented like this: https://github.com/gampleman/elm-visualization/blob/e0dfaf17bb719296b3b4f5112fef3dc72c4f31f8/src/Visualization/Scale/Colors.elm#L47-L59

They were originally taken from D3, and due to the differing implementation I am not sure if these are indeed the same. But if they are, I think the reasoning still applies - categorical color palletes we provide should have good accessibility properties out of the box.

Reply:
They are probably the same and thinking about it you might be right, a 20 colours categorical palette would never be very accessible. Too many colours. 
So ideally we are looking for a good set of short categorical colour schemes. Will have a thought on this.

Reply:
As a thought, we could use the categorical schemes (Accent, Dark2, Paired, Pastel1, Pastel2, Set1, Set2, Set3) in [colorbrewer](https://colorbrewer2.org/export/colorbrewer.js) to create equivalent palette generators in elm. So we stick to the maximum number of colours per scheme in colorbrewer that is 12. These are also in vega, but instead of using a static palette, we could implement a palette generator that takes the number of colors as an argument.
Then maybe we could look for other categorical palettes worth considering.

Reply:
Ignore the comment above. Just realized that the colours in the categorical schemes from colorbrewer repeat themselves, so just one declaration is needed. I removed the 20 categorical ones and now we are left with most of the [colorbrewer](https://colorbrewer2.org) ones already. Next step I will have a look for some more schemes.